src/main/webapp/resources/static/silent-check-sso.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Silent SSO Check</title>
+  </head>
+  <body>
+    <script>
```

Comment:
Not sure when we need a nonce but in my test I included it:

`<#assign wp=JspTaglibs[ "/aps-core"]>
<html>
<body>
<script nonce="<@wp.cspNonce />">
    parent.postMessage(location.href, location.origin)
</script>
</body>
</html>`

Reply:
@nshaw the problem is this is a static file. I can't do freemarker stuff in there. quick solution would be to make a page via appbuilder for that but it's not a good solution in my opinion (since there's a chance that this would be modified by a user). I would allow this for now but there should be a technical debt for this. Best solution for this should be this particular page will be generated via app-engine (admin console). Maybe a backend person can answer this? another thing I can think of is maybe would it be okay to omit nonce? what do you think of this @w-caffiero-entando ?

Reply:
Ah, makes sense. I built it as an actual Page in Entando which would be an alternative here, if we need the nonce. Hopefully @w-caffiero-entando can clarify what's needed.